pkg/util/annotation.go

```diff
@@ -5,41 +5,50 @@ import (
 )
 
 func GetRegion(labels map[string]string) (string, bool) {
-	if _, ok := labels[v1.LabelTopologyRegion]; ok { // Label as of 1.17
+	_, ok := labels[v1.LabelTopologyRegion]
```

Comment:
there is no need to make such change as the original code line is a better approach to me, it create a variable in smaller scope and the memory would be release soon. and it's a common practice in golang.
`if _, ok := labels[v1.LabelTopologyRegion]; ok { `

Reply:
> there is no need to make such change as the original code line is a better approach to me, it create a variable in smaller scope and the memory would be release soon. and it's a common practice in golang. `if _, ok := labels[v1.LabelTopologyRegion]; ok { `

The original code has a little bit much `if-else`, which seems is not necessary, so the pr is try to reduce the unnecessary code for making the logic clearn.
Speaking of golang's idioms, on one hand I suppose `happy path` is the commonly used way in golang for reducing indent and embeded codes, on the other hand the memory will be eventually released after the function returned, just sooner or later, not a big deal.

Reply:
agreed with @mfanjie

Reply:
> agreed with @mfanjie

updated.